server/src/ClientToDBAPI/IntersectionsRoute/mainRoute.ts

```diff
@@ -93,25 +92,38 @@ intersectionsRoute.post('/createContactEvent', (request: Request, response: Resp
     const newEvent = convertEventToDBType(request.body);
     graphqlRequest(CREATE_CONTACT_EVENT, response.locals, {contactEvent: JSON.stringify(newEvent)})
     .then(result => {
-        console.log(result)
-        response.send(result)
+        console.log(result);
+        response.send(result);
         })
     .catch(err => {
         console.log(err);
-        response.status(errorStatusCode).send('error in fetching data: ' + err)
+        response.status(errorStatusCode).send('error in creating event: ' + err);
     });
 });
 
 intersectionsRoute.post('/updateContactEvent', (request: Request, response: Response) => {
     const updatedEvent = convertEventToDBType(request.body);
     graphqlRequest(EDIT_CONTACT_EVENT, response.locals, {event: JSON.stringify(updatedEvent)})
     .then(result => {
-        console.log(result)
-        response.send(result)
+        console.log(result);
+        response.send(result);
+    })
+    .catch(err => {
+        console.log(err);
+        response.status(errorStatusCode).send('error in updating event: ' + err);
+    });
+});
+
+intersectionsRoute.delete('/deleteContactEvent', (request: Request, response: Response) => {
+    const contactEventId = +request.query.contactEventId;
```

Comment:
https://stackoverflow.com/questions/51069552/axios-delete-request-with-body-and-headers

Maybe change it to body